src/main/java/com/example/demo/service/BattleService.java

```diff
@@ -41,11 +47,23 @@ private List<Battle> findBattlesEndWithinPerm(int perm) {
 		return battleRepository.findByStatusAndUpdatedAtBetween(BattleStatus.END, lastBattleDay, now);
 	}
 
-	public BattleDetailsListResponseDto getBattleDetailsListInProgress() {
+	public BattleDetailsListResponseDto getBattleDetailsListInProgress(Principal principal) {
+		Member member = principalService.getMemberByPrincipal(principal);
 		List<Battle> battleListInProgress = battleRepository.findAllByStatusEquals(BattleStatus.PROGRESS);
+		List<Vote> votes = voteRepository.findAllByVoterId(member.getId());
+
+		List<Battle> notVotedBattleListInProgress = battleListInProgress.stream()
+			.filter(battle -> votes.stream()
+				.noneMatch(vote -> checkVotedBattle(vote, battle))
```

Comment:
checkVotedBattle을 private 메소드로 빼니, 이해하기 힘든 것 같습니다.
차라리 제가 앞서언급한것 처럼 바꾸고
```suggestion
				.noneMatch(vote -> vote.hasBattle(battle.getId())
```
처럼 하는건 어떤가요?